azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -204,6 +204,34 @@ public Project getProject(int id) {
     return projectsById.get(id);
   }
 
+    /**
+     * fetch inactive project from db by project_id
+     *
+     * @param name
+     * @return
+     */
+    public Project getInactiveProject(String name) {
```

Comment:
Yes, I named it InActive as there was already a method getProject to fetch active projects from Cache and to minimize code refractoring.
I have merged two methods and added another method to test if a project is Active.